### ISSUE TITLE
broken on 10.0.4

### ISSUE THREAD

**Issue by yaolet** [mart1nro/joycontrol#65]
with 10.0.4 firmware, the emulation will be disconnected on leaving the Change grip/order screen.

tested with rpi & x86 ubuntu 20.04

**Comment by mart1nro** [comment]
I tested it with 10.0.4 on my Laptop and it It still works.

I'm going to check it on my Raspberry Pi 4 as well once I have time and report back.

**Comment by dpxcc** [comment]
duplicate to https://github.com/mart1nro/joycontrol/issues/26?

**Comment by yaolet** [comment]
> duplicate to #26?

yeah seems like the same issue.  Its very odd that the same pi worked last week with the same code. Suddenly it just stopped working

**Comment by yaolet** [comment]
tested again on 10.1.0 on x86 ubuntu 20.04

problem still exists

**Comment by Poohl** [comment]
When did this problem start to appear? I'm suddenly having the same problem on ubuntu 20.04 since 2 weeks ago (using a laptop). The odd thing is the connection only drops after you send the first input, until then it's totally fine (but useless).

Ubuntu version: 20.04
Switch version: 11.0.1
